### ISSUE TITLE
Approaching/Exceeded Limits Emails

### ISSUE THREAD

**Issue by benhaynes** [directus/cloud-beta#12]
We should send emails for when a project is approaching its plan limits, and after it has exceeded its limits. Both email templates are already ready to go in SendGrid.

**Comment by shealavington** [comment]
On the note of limits, I wasn't aware that my limits would be used simply by navigating the Directus APP, I somewhat expected my limits to be the external requests that I make. I managed to rack up 300 odd requests just setting up some tables and basic actions yesterday. 

Granted 2mil requests should be enough, but perhaps it's not clear as a user, that using the "backend" (app) would eat my limits too.

**Comment by benhaynes** [comment]
Yeah, we had a conversation about this earlier. I still think it's a good idea, but we should make that clear. My reasoning:

If you spin up a Cloud project as a Project Management tool, you wouldn't be connecting to the API at all. I think we can basically give enough base requests to accommodate this... but it's a good "catch" so no one abuses the system. :)

Thoughts?

Also, this reminds me that "items count" would also include the `directus_*` tables... reason being:

If you are using a Cloud project for IoT data... you might _update_ a small number of items (<100) with tens of thousands of revisions (the IoT device data/value updates) and that has a big impact on our system. Again, with normal usage this shouldn't be an issue, but we should make this very clear in the legal. Another option for this would be to have a separate "revision" usage limit (with dedicated overages).

Thoughts???

**Comment by rijkvanzanten** [comment]
> Also, this reminds me that "items count" would also include the directus_* tables... reason being:

If that's the case, we should bump the item count by a lot. directus_activity fills up incredibly quickly

**Comment by benhaynes** [comment]
The more I think about it, I think we should have a separate technical limit with overages for item revision. They are very different from user generated items and it will be too confusing to lump them together.
